app/routes/app/$type/new.tsx

```diff
@@ -63,6 +63,8 @@ export default function CreateMarketplace() {
         <ValidatedForm<LaborMarketNew> validator={validator} method="post" defaultValues={defaultValues}>
           <h1 className="text-3xl font-semibold antialiased">Create Challenge Marketplace</h1>
           <MarketplaceForm projects={projects} tokens={tokens} />
+          <input type="hidden" name="userAddress" value={user?.address} />

```

Comment:
Should set this data in the backend instead?

Reply:
yeah... back to what I had before